build/Windows.Toolkit.Common.props

```diff
@@ -0,0 +1,46 @@
+<Project>
+
+  <PropertyGroup>
+    <Company>.NET Foundation</Company>
+    <Authors>Microsoft.Toolkit</Authors>
+    <Product>Windows Community Toolkit</Product>
+    <PackageLicenseExpression>MIT</PackageLicenseExpression>
+    <PackageRequireLicenseAcceptance>true</PackageRequireLicenseAcceptance>
+    <Copyright>(c) .NET Foundation and Contributors.  All rights reserved.</Copyright>
+    <PackageProjectUrl>https://github.com/windows-toolkit/WindowsCommunityToolkit</PackageProjectUrl>
+    <PackageReleaseNotes>https://github.com/windows-toolkit/WindowsCommunityToolkit/releases</PackageReleaseNotes>
+    <PackageIcon>nuget.png</PackageIcon>
+    <PackageIconUrl>https://raw.githubusercontent.com/windows-toolkit/WindowsCommunityToolkit/master/build/nuget.png</PackageIconUrl>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <Features>Strict</Features>
+    <Nullable>Disable</Nullable>
+    <LangVersion>Latest</LangVersion>
+    <DefaultLanguage>en-US</DefaultLanguage>
+    <NoPackageAnalysis>true</NoPackageAnalysis>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <TargetPlatformBaseVersion>10.0</TargetPlatformBaseVersion>
+    <TargetPlatformRevision>19041</TargetPlatformRevision>
+    <TargetPlatformMinRevision>17763</TargetPlatformMinRevision>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <IsDesignProject>$(MSBuildProjectName.Contains('.Design'))</IsDesignProject>
+    <IsSampleProject>$(MSBuildProjectName.Contains('.Sample'))</IsSampleProject>
+    <IsTestProject>$(MSBuildProjectName.Contains('Test'))</IsTestProject>
+    <IsUwpProject>$(MSBuildProjectName.Contains('.Uwp'))</IsUwpProject>
+    <IsUwpProject Condition="$(IsDesignProject)">False</IsUwpProject>
+    <IsCoreProject Condition="$(IsDesignProject) or $(IsSampleProject) or $(IsTestProject)">False</IsCoreProject>
+    <IsCoreProject Condition="'$(IsCoreProject)' == ''">True</IsCoreProject>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <None Include="$(BuildToolsDirectory)nuget.png" Pack="true" PackagePath="\" />
```

Comment:
QQ: Any reason you moved this back to the root instead of in it's own directory?

Reply:
Consistency! Every other "official" .NET package has it's icon in the root. Also, there's an another reason why, but I couldn't find that discussion.

Reply:
Cool, looks like they call it `Icon.png` in most cases, think we should do the same then?

Reply:
I'm okay with that but is it safe to rename the file on repo or do we change when packing?

```suggestion
    <None Include="$(BuildToolsDirectory)nuget.png" Pack="true" PackagePath="\Icon.png" />
```

Since, there would be links in the wild referring back to the file!

Reply:
Good point, I'm not sure if we directly linked to this in the past, maybe as we used to use the icon url... for now renaming on package seems like a good solution. Just have to update the `PackageIcon` tag too, eh?

Reply:
Yes.